base/applications/sndrec32/sndrec32.cpp

```diff
@@ -619,8 +598,12 @@ WndProc(HWND hWnd,
                     break;
 
                 case ID_ABOUT:
-                    DialogBox(hInst, MAKEINTRESOURCE(IDD_ABOUTBOX), hWnd, AboutDlgProc);
-                    return TRUE;
+                    LoadStringW(hInst, IDS_APP_TITLE, szAppName, sizeof(szAppName) / sizeof(WCHAR));                      
+                    LoadStringW(hInst, IDS_APP_AUTHORS, szAppAuthors, sizeof(szAppAuthors) / sizeof(WCHAR));
```

Comment:
Why load an empty string and pass it to ShellAbout, instead of passing `NULL`?

Reply:
I initially populated "IDS_APP_AUTHORS" with ReactOS Team, then got the comment that it was not useful. Therefore I set it to an empty string but kept the code, in case with later have a proper Author to quote for this module.

Would you prefer to have this part of code removed, replaced by NULL and IDS_APP_AUTHORS definition discarded as well ?

Reply:
Yes, adding dead code to the codebase 'in case someone wants to change something later' is not really a good idea.

Reply:
OK, i'll refactor.

Reply:
Done !